app/test_split_integer.py

```diff
@@ -2,20 +2,29 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    value = 10
+    parts = split_integer(10, 3)
+    assert sum(parts) == value
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+    value = 12
+    number_of_parts = 4
+    result = split_integer(value, number_of_parts)
+    expected = [3, 3, 3, 3]
+    assert result == expected
```

Comment:
The expected result `[3, 3, 3, 3]` is incorrect for splitting 12 into 4 parts. The correct expected result should be `[3, 3, 3, 3]` as the value 12 is perfectly divisible by 4. Please verify the logic in the `split_integer` function to ensure it returns the correct result.